wrappers/MathCAD/Prime/README.rst

```diff
@@ -1,19 +1,45 @@
-CoolProp for MathCAD Prime
-==========================
+CoolProp Wrapper for MathCAD Prime 3.x or later
+==================================================
 
-Info
-----
+| Copyright Scott Polak and Ian Bell, 2013
+| Updated by Jeff Henning, 2016
 
-Wrapper built by Scott Polak (scott.polak@aei.com), December 2013
+Prerequisites
+==============
 
+* You will need to have Visual Studio 2008 installed.  Alternatively newer versions of Microsoft Visual Studio C++ should be fine.  Unfortunately, you will need the professional version of Visual Studio C++ as Mathcad Prime libraries are 64-bit and require the 64-bit compiler; the Express version of Visual Studio will not work.
```

Comment:
Or you could install Visual Studio 2013+ express, which includes 64-bit compilation.  I use/like Visual Studio 2015 Express.